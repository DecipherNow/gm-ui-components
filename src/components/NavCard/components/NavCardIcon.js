import styled from "styled-components";

const NavCardIcon = styled.span`
  display: flex;
  align-items: center;
`;

export default NavCardIcon;
