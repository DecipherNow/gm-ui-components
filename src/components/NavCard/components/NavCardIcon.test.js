import React from "react";

import NavCardIcon from "./NavCardIcon";

describe("NavCardIcon", () => {
  it("matches snapshot", () => {
    const aNavCardIcon = shallow(<NavCardIcon />);
    expect(aNavCardIcon).toMatchSnapshot();
  });
});
