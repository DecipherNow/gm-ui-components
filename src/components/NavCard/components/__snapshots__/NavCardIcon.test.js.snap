// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavCardIcon matches snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "NavCardIcon-vgwdqc-0",
        "isStatic": true,
        "rules": Array [
          "display:flex;align-items:center;",
        ],
      },
      "displayName": "NavCardIcon",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "NavCardIcon-vgwdqc-0",
      "target": "span",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
